pkg/apb/executor.go

```diff
@@ -201,3 +200,22 @@ func checkPullPolicy(policy string) (v1.PullPolicy, error) {
 
 	return value, nil
 }
+
+func copySecretsToNamespace(executionContext ExecutionContext,
```

Comment:
Nit: We have a couple stylistic patterns for long function signatures in the codebase right now. Admittedly, it's not super consistent, but we should try to not introduce other forms as well IMO. Personally, I like a vertically favored format:

```
func copySecretsToNamespace(
	executionContext ExecutionContext,
	clusterConfig ClusterConfig,
	k8scli *clientset.Clientset,
	secrets []string,
) error { // Return values
	// Implementation
}
```

Also a longer form format, wrapping after the last full var/type crosses something like 80 chars:

```
func copySecretsToNamespace(executionContext ExecutionContext, clusterConfig ClusterConfig,
	k8scli *clientset.Clientset, secrets []string) error {
	// Implementation
}
```

I would just advocate picking one and sticking with it.